src/Sql/Insert.php

```diff
@@ -160,7 +160,8 @@ public function getRawState($key = null)
         $rawState = [
             'table' => $this->table,
             'columns' => array_keys($this->columns),
-            'values' => array_values($this->columns)
+            'values' => array_values($this->columns),
```

Comment:
Thank you for this great dialog, @alextech!  You are definitely **not** "coming too strong against" my thoughts on these topics - I have learned quite a bit plus you have given me a different perspective on writing APIs.  Very good stuff!

As I am new to being involved with large, public projects like ZF2, I was unsure what we do 'next' in our dialog.  I am glad you suggested we let @weierophinney be the judge - good idea!

Until the next PR :-)